src/Umbraco.Web.UI.Client/src/views/components/content/umb-content-node-info.html

```diff
@@ -19,6 +19,22 @@
                 </ul>
             </umb-box-content>
         </umb-box>
+        <umb-box data-element="node-info-redirects">
+            <umb-box-header title-key="redirectUrls_redirectUrlManagement"></umb-box-header>
+            <umb-box-content class="block-form">
+                <div style="position: relative;">
+                    <div ng-if="loadingRedirectUrls" style="background: rgba(255, 255, 255, 0.8); position: absolute; top: 0; left: 0; right: 0; bottom: 0;"></div>
+                    <umb-load-indicator ng-if="loadingRedirectUrls"></umb-load-indicator>
```

Comment:
Once loaded and there are redirects, it would be good to show a one-liner like: 

> The following URLs are pointing to the current page:

It makes it more clear what this is, the tiny arrow pointing inwards is pretty easy to overlook. 😉

Reply:
I have added this, ironically I had it in originally, but I always worry when I'm adding text explaining what something does, because it sort of means it isn't obvious from the UX, and it's more text to translate... perhaps it should be an item on the backoffice tour?